UIKitDZ/UIKitDZ/Menu screen/MenuViewController.swift

```diff
@@ -1,17 +1,212 @@
+    private lazy var rectangleOne: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 20, y: 216, width: 335, height: 80))
+        imageView.image = UIImage(named: "rectangleImage")
+        return imageView
+    }()
+
+    private lazy var ptiPateLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 25, y: 30, width: 180, height: 19))
+        label.text = "Пти пате aля «РюсЪ»"
+        label.textColor = .black
+        label.font = UIFont(name: "Verdana-BoldItalic", size: 14.0)
+        label.textAlignment = .left
+        return label
+    }()
+
+    private lazy var cakeImage: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 272, y: 221, width: 70, height: 70))
+        imageView.image = UIImage(named: "cakeImage")
+        return imageView
+    }()
+
+    // вторая позиция меню
+    private lazy var rectangleTwo: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 20, y: 316, width: 335, height: 80))
+        imageView.image = UIImage(named: "rectangleImage")
+        return imageView
+    }()
+
+    private lazy var hotBeveragesLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 25, y: 30, width: 180, height: 19))
+        label.text = "Горячiя напитки"
+        label.textColor = .black
+        label.font = UIFont(name: "Verdana-BoldItalic", size: 14.0)
+        label.textAlignment = .left
+        return label
+    }()
+
+    private lazy var capImage: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 252, y: 15, width: 70, height: 48))
+        imageView.image = UIImage(named: "capCoffeeImage")
+        return imageView
+    }()
+
+    // третья позиция/ кнопка
+    private lazy var rectangleThree: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 20, y: 416, width: 335, height: 80))
+        imageView.image = UIImage(named: "rectangleImage")
+        return imageView
+    }()
+
+    private lazy var capSpoonImageView: UIImageView = {
+        let imageView = UIImageView(frame: CGRect(x: 252, y: 5, width: 70, height: 70))
+        imageView.image = UIImage(named: "capSpoonImage")
+        return imageView
+    }()
+
+    private lazy var coffeeLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 25, y: 30, width: 180, height: 19))
+        label.text = "Кофий"
+        label.font = UIFont(name: "Verdana-BoldItalic", size: 14.0)
+        label.textAlignment = .left
+        label.textColor = .black
+        return label
+    }()
+
+    private lazy var coffeeButton: UIButton = {
+        let button = UIButton(frame: CGRect(
+            x: 0,
+            y: 0,
+            width: rectangleThree.frame.width,
+            height: rectangleThree.frame.height
+        ))
+        button.backgroundColor = .clear
+        button.addTarget(self, action: #selector(coffeeButtonTapped), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Life cycle
 
     override func viewDidLoad() {
```

Comment:
```suggestion
    override func viewDidLoad() {
```
в этом методе только методы вызываем